pkg/cluster/certs.go

```diff
@@ -66,7 +66,7 @@ func (c *Controller) RotateAllCerts(ctx context.Context) error {
 		c.Log.Warnf("Failed to delete rotate pods: %v", err)
 	}
 	opts := metav1.ListOptions{
-		LabelSelector: PrimaryRoleLabel,
+		LabelSelector: "node-role.kubernetes.io/master=,node-role.kubernetes.io/control-plane=",
```

Comment:
this will not work if it does not have both labels. i havent figured out how to query for either key. you may have to make two requests here.

Reply:
if PrimaryRoleLabel is no longer used can you remove it?

Reply:
i just made two queries and tested this as well.